shared/src/ledger/native_vp/ethereum_bridge/authorize.rs

```diff
@@ -1,37 +1,43 @@
 //! Functionality to do with checking whether a transaction is authorized by the
 //! "owner" of some key under this account
-use eyre::Result;
-use namada_core::types::address::Address;
+use std::collections::BTreeSet;
 
-use crate::ledger::native_vp::StorageReader;
+use namada_core::types::address::Address;
 
+/// For wrapped ERC20 transfers, checks that `verifiers` contains the `sender`'s
+/// address - we delegate to the sender's VP to authorize the transfer (for
+/// regular Namada accounts, this will be `vp_implicit` or `vp_user`).
 pub(super) fn is_authorized(
-    _reader: impl StorageReader,
-    _tx_data: &[u8],
-    _owner: &Address,
-) -> Result<bool> {
-    tracing::warn!(
-        "authorize::is_authorized is not implemented, so all transfers are \
-         authorized"
-    );
-    Ok(true)
+    verifiers: &BTreeSet<Address>,
+    sender: &Address,
+) -> bool {
+    verifiers.contains(sender)
```

Comment:
honest question, I've always wondered if we trigger the receiver vp as well. Or can people not refuse money from certain accounts?

Reply:
It seems we should trigger the receiver account as well.

Reply:
Will add this, will require some small refactoring.